jupyterhub/templates/NOTES.txt

```diff
@@ -35,4 +35,11 @@ custom:
 WARNING: You are using user placeholders without pod priority enabled, either
 enable pod priority or stop using the user placeholders to avoid wasting cloud
 resources.
+{{- end }}
+
+{{- if .Values.hub.uid }}
+
+DEPRECATION: hub.uid is deprecated in jupyterhub chart 0.9. Set the hub.containerSecurityContext.runAsUser value
+directly instead.
+
```

Comment:
There is an edge case you about uid being zero for the if statement.

Reply:
Good catch, fixed.